plugins/pulp_rpm/plugins/importers/yum/sync.py

```diff
@@ -269,7 +269,12 @@ def run(self):
             finally:
                 # clean up whatever we may have left behind
                 shutil.rmtree(self.tmp_dir, ignore_errors=True)
-            self.save_repomd_revision()
+
+            if self.config.override_config.get(importer_constants.KEY_FEED):
+                self.erase_repomd_revision()
+            else:
+                self.save_repomd_revision()
```

Comment:
It looks like a lot of the sync tests are commented out via `@skip_broken`, I am not sure if there are any tests for `run()` that are operational:

```
$ nosetests -s plugins/test/unit/plugins/importers/yum/test_sync.py
SSSS..SSSSSSSSSSSSSSSSS.SSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSS
----------------------------------------------------------------------
Ran 105 tests in 0.534s

OK (SKIP=102)
```